lib/config/options/index.ts

```diff
@@ -1968,12 +1968,22 @@ const options: RenovateOptions[] = [
       commitMessageSuffix: '[SECURITY]',
       branchTopic: `{{{datasource}}}-{{{depNameSanitized}}}-vulnerability`,
       prCreation: 'immediate',
+      vulnerabilityFixStrategy: 'lowest',
     },
     mergeable: true,
     cli: false,
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'vulnerabilityFixStrategy',
+    description:
+      'Strategy to use when fixing vulnerabilities. `lowest` will use the lowest fixed version, `highest` will use the highest fixed version.',
```

Comment:
Personally, I find "highest fixed version" not very intuitive. Suggestion:

```suggestion
      'Strategy to use when remediating vulnerabilities. `lowest` will propose the earliest version with a fix, `highest` will always pick the latest version.',
```

Reply:
Do you think "remediation" is commonly understood amongst normal developers? I use it a lot because of Mend but I was preferring "vulnerability fix" instead of "remediation" to make it more intuitive to non-security folk

Reply:
You're right, for general clarity, "fixing vulnerabilities" is likely the better choice 👍